nibabel/nicom/tests/test_dicomwrappers.py

```diff
@@ -386,19 +388,28 @@ def fake_frames(seq_name, field_name, value_seq):
     class Fake:
         pass
 
-    frames = []
-    for value in value_seq:
-        fake_frame = Fake()
+    if frame_seq == None:
+        frame_seq = [Fake() for _ in range(len(value_seq))]
```

Comment:
`is None` is the more common check.

```suggestion
    if frame_seq is None:
        frame_seq = [Fake() for _ in range(len(value_seq))]
```

Reply:
Fixed